packages/db/prisma/schema.prisma

```diff
@@ -81,6 +99,62 @@ model PreApprovedUser {
     @@map("pre_approved_users")
 }
 
+// Group model - collection of users with ownership and management
+model Group {
+    id          Int      @id @default(autoincrement())
+    name        String
+    description String?
+    created_at  DateTime @default(now())
+    updated_at  DateTime @updatedAt
+
+    // Group ownership
+    owner    User @relation("GroupOwner", fields: [owner_id], references: [id], onDelete: Cascade)
+    owner_id Int
+
+    // Relations
+    managers GroupManager[]
+    members  GroupMember[]
+    
+    // Project sharing relations
+    sharedProjects ProjectGroupShare[]
+
+    @@map("groups")
+}
+
+// Group managers - users who can manage the group (but are not owners)
+model GroupManager {
+    id       Int @id @default(autoincrement())
+    group_id Int
+    user_id  Int
+
+    created_at DateTime @default(now())
+
+    // Relations
+    group Group @relation(fields: [group_id], references: [id], onDelete: Cascade)
+    user  User  @relation(fields: [user_id], references: [id], onDelete: Cascade)
+
+    // Ensure a user can only be a manager of a group once
+    @@unique([group_id, user_id])
+    @@map("group_managers")
+}
+
+// Group members - regular users in the group
+model GroupMember {
```

Comment:
hmm, I'd probably have one GroupMembership table and a column for the membership_type (manager|member). Seems awkward for the table itself to dictate the kind of membership. But maybe you had a reason for the separation?